udah ada splash dan welcome screen tinggal rapihkan semua

muhturmudzi opened the pull request.

muhturmudzi committed:
udah ada splash dan welcome screen tinggal rapihkan semua

muhturmudzi committed:
tinggal rapihin bray

seagalputra commented:
@muhturmudzi itu masih salah di welcome activitynya, harusnya intentnya bukan ke LoginActivity.class tapi ke MainActivity.class

muhturmudzi committed:
splash screen ok
welcome screen ok
login screen bingung mau ditambah apalagi, trus feedback nya bakal melayang kalo kita klik edit text, masalahnya edit text nya udah ke pencet mulu
message screen belum ditambah empty state kalo message nya gaada, masih belum dapet cara nya

seagalputra committed:
Merge branch 'bappeda-1.3' into bappeda-1.3
